src/components/Step1/Step1Form.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { FC } from 'react'
+import { useSelector } from 'react-redux'
+import { TextInput, UPLOAD_STATUS } from 'kyb-core'
+import { useIntl } from 'react-intl'
+import SelectInput from '../FormGroup/SelectInput'
+import { CompanyTypeOptions, LegalFormTypeOptions } from './Options'
+import StepNavButtons from '../StepNavButtons/StepNavButtons'
+import Upload from '../Upload/Upload'
+import FirstUploadLabel from './FirstUploadLabel'
+import SecondUploadLabel from './SecondUploadLabel'
+import { useStep1Hook } from '../../hooks/useStep1Hook'
+import { useStep1FormHook } from '../../hooks/useStep1FormHook'
+import { IFileState } from '../../interfaces/Upload'
+import { UPLOAD_ACTIONS } from '../../services/Upload/Upload'
+import { IStore } from '../../interfaces/Store'
+
+const initialState: IFileState[] = [
+  {
+    file: null,
+    message: '',
+    personTypeId: '1', // TODO update when we have the correct type
+    status: 'getFile',
+    typeId: '1', // TODO update when we have the correct type
+  },
+  {
+    file: null,
+    message: '',
+    personTypeId: '2', // TODO update when we have the correct type
+    status: 'getFile',
+    typeId: '1', // TODO update when we have the correct type
+  },
+]
+
+const Step1Form: FC = () => {
+  const intl = useIntl()
+  const textInputLabel = intl.formatMessage({
+    id: 'form.field.companyType.companyName.label',
+  })
+  const { dispatch, firstFile, onChange, secondFile, state } = useStep1Hook(
+    initialState
+  )
+  const {
+    companyType,
+    form: {
+      control,
+      handleSubmit,
+      formState: { errors },
+      register,
+      trigger,
+    },
+    onSubmit,
+    updateCompanyType,
+  } = useStep1FormHook()
+  /* eslint no-shadow: ["error", { "allow": ["state"] }] */
+  const {
+    companyName: savedCompanyName,
+    companyType: savedCompanyType,
+  } = useSelector((state: IStore) => state.folderReducer.folder)
+  // TODO: remove last "&& false" check as it is used only to force the happy flow
+  const requireBasicCompanyDetails: boolean =
+    (!savedCompanyName || !savedCompanyType) && false
+
+  const validateUploadedFiles = () => {
+    let filesAreUploaded = true
+    state.forEach((fileState, index) => {
+      if (
```

Comment:
```suggestion
      const UPLOADED_STATUSES = [UPLOAD_STATUS.pending, UPLOAD_STATUS.validated];
      if (!UPLOADED_STATUSES.includes(fileState.status)) {
      ...
```
Maybe the constant UPLOADED_STATUSES should be set somewhere global, because I think this check should be done on step 2 also.
Even the check can be moved into a function to be used globally
```
// 
// src/services/utils.ts
export const checkIfFileIsUploaded = (status:string) => UPLOADED_STATUSES.includes(fileState.status)

//src/components/Step1/Step1Form.tsx
import {checkIfFileIsUploaded} from '../../services/utils.ts'
...
if (!checkIfFileIsUploaded(fileState.status)) {
```

Reply:
Set UPLOADED_STATUSES as a global variable to be used.